mm2src/coins/z_coin/storage/walletdb/wasm/mod.rs

```diff
@@ -744,7 +746,7 @@ mod wasm_test {
     //    async fn create_to_address_fails_on_unverified_notes() {
     //        // init blocks_db
     //        let ctx = mm_ctx_with_custom_db();
-    //        let blockdb = BlockDbImpl::new(&ctx, TICKER.to_string(), PathBuf::new()).await.unwrap();
+    //        let blockdb = BlockDbImpl::new(&ctx, TICKER.to_string(), PathBuf::new(), None).await.unwrap();
```

Comment:
There is a todo above `Todo: Uncomment after improving tx creation time`
could you tell what the problem with this? 
I remember that there was an issue with blocks synchronisation, which was fixed by adding sync params. but what  about tx creation time?

Reply:
generating zcoin txs uses lot of resource and takes time to run...there're some optimization for native targets that cannot be achieved with WASM yet